tests/token/test_token_init.py

```diff
@@ -0,0 +1,12 @@
+
+
+def test_erc20_deploy(w3, SNT_token):
+    a0 = w3.eth.accounts[0]
+    assert SNT_token.name() == "Status Network Token"
+    assert SNT_token.symbol() == "SNT"
+    assert SNT_token.decimals() == 18
+    assert SNT_token.total_supply() ==  3470483788*10**18
```

Comment:
@andytudhope these type of assert being super clean, is what I like the most of pytest / web3py testing :)